src/components/CrossButton.tsx

```diff
@@ -0,0 +1,36 @@
+import cx from 'classnames';
+
+type CrossButtonProps = {
+  onClick: () => void;
+  className?: string;
+};
+
+export const CrossButton = ({ onClick, className }: CrossButtonProps) => (
+  <button
+    type="button"
+    aria-label="Zavřít"
+    className={cx('group', className)}
```

Comment:
What is the group doing here? https://tailwindui.com/components/application-ui/elements/button-groups

 Is this a group?